real_estate/models/property.py

```diff
@@ -0,0 +1,112 @@
+from datetime import timedelta
+
+from odoo import api, exceptions, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.float_utils import float_compare
+
+
+class Property(models.Model):
+    _name = "property"
+    _description = "Estate Property Model"
+    _order = "id desc"
+
+    name = fields.Char("Title", required=True)
+    description = fields.Text("Description")
```

Comment:
if the name is the same you don't need to put it
```suggestion
    description = fields.Text()
```